courses/free-courses-en.md

```diff
@@ -765,6 +765,7 @@
 
 ### Graph Theory
 
+* [Graph Theory](https://www.youtube.com/watch?v=E40r8DWgG40&list=PLEAYkSg4uSQ2fXcfrTGZdPuTmv98bnFY5) - IISER Pune 
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes